### PR TITLE
correct ks units, clean up comments, bug fix for BL99 enthalpy calculation

### PR DESCRIPTION
**Pull request by eclare108213**
- [x] Short (1 sentence) summary of your PR: 
Bug fix for BL99 and minor cleanup:
- correct ks units in dEdd code
- clean up comment statements
- uncomment enthalpy calculation when temperature is adjusted, for BL99 thermo (bug fix)

- [x] Developer(s): 
@eclare108213 
- [x] Suggest PR reviewers from list in the column to the right.
- [x] Please copy the PR test results link or provide a summary of testing completed below.
    base_suite is BFB:
```
196 measured results of 196 total results
196 of 196 tests PASSED
0 of 196 tests PENDING
0 of 196 tests MISSING data
0 of 196 tests FAILED
```
It's likely that results with BL99 thermo will not be BFB if/when the bug fix condition is hit.

- How much do the PR code changes differ from the unmodified code? 
    - [x] bit for bit except for BL99 bug fix (occasionally)
    - [ ] different at roundoff level
    - [ ] more substantial 
- Does this PR create or have dependencies on CICE or any other models?
    - [ ] Yes
    - [x] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [x] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/.)
    - [ ] Yes
    - [x] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [x] No 
- [x] Please provide any additional information or relevant details below:
Closes https://github.com/CICE-Consortium/Icepack/issues/450

### CODE REVIEW

**Comment by apcraig** [comment]
This looks fine.  Would you like me to run a regression test suite?

**Comment by eclare108213** [comment]
I have a base_suite regression test running now - looks good so far.

**Comment by eclare108213** [comment]
base_suite came back fully BFB, although results could change in longer runs with BL99.  Ready for review/merge.

File: columnphysics/icepack_therm_vertical.F90 [hunk 1]
```diff
@@ -970,13 +970,12 @@ subroutine init_vertical_profile(nilyr,    nslyr,    &
...
-!                   zTin(k) = -zSin(k)*depressT - puny
-!                   zqin(k) = -rhoi*cp_ocn*zSin(k)*depressT
-!               endif
-!         endif
+         if (ktherm == 1) then
+               if (zTin(k)>= -zSin(k)*depressT) then
+                   zTin(k) = -zSin(k)*depressT - puny
+                   zqin(k) = -rhoi*cp_ocn*zSin(k)*depressT
+               endif
+         endif
```
Comment (apcraig) [inline columnphysics/icepack_therm_vertical.F90:978]: indentation could be fixed here.

Comment (eclare108213) [reply columnphysics/icepack_therm_vertical.F90:976]: done

Comment (apcraig) [reply columnphysics/icepack_therm_vertical.F90:976]: thank you.

**Pull request merged**